includes/class-wc-order-factory.php

```diff
@@ -100,8 +100,12 @@ public static function get_order_item( $item_id = 0 ) {
 				case 'tax' :
 					$classname = 'WC_Order_Item_Tax';
 				break;
+				default :
+					$classname = apply_filters( 'woocommerce_get_order_item_classname', $classname, $item_type, $item_id );
```

Comment:
Yes you're right, I initially thought it could be set to false, but after a second look that is only in weird cases